main/libcamera/patches/02-disable-ipa-signing.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/meson.build
```

Comment:
you shouldn't disable the signing

i'm not sure how to make it run post-debugsplit though (stripping the binary after signing breaks the signature etc)

Reply:
we could expose stripping as a utility module and then it could be done in post_install

Reply:
or we could include affected files under `nostrip_files`, ig..

Reply:
no, i don't want that